src/host/VtIo.cpp

```diff
@@ -534,33 +534,30 @@ static char* formatAttributes(char* out, WORD attributes) noexcept
     out += 3;
 
     // 2 bytes.
-    if (attributes & COMMON_LVB_REVERSE_VIDEO)
+    if (attributes.IsReverseVideo())
     {
         memcpy(out, ";7", 2);
         out += 2;
     }
 
-    // `attributes` of exactly `FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED`
-    // are often used to indicate the default colors in Windows Console applications.
-    // Since we always emit SGR 0 (reset all attributes), we simply need to skip this branch.
-    //
-    // 7 bytes (";97;107").
-    if ((attributes & (FG_ATTRS | BG_ATTRS)) != (FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_RED))
+    // 7 bytes (";97").
+    if (attributes.GetForeground().IsIndex16())
+    {
+        out = fmt::format_to(out, FMT_COMPILE(";{}"), attributes.GetForeground().GetIndex());

```

Comment:
I don't think this works. Doesn't this turn into `;7` (etc)?

Reply:
Ugh, right.
Easy live? ❌